### PR TITLE
Remove the copyright from the samples files

### LINKED ISSUE #1576: Remove the copyright from the samples files
There is no need for copyright in the docs/Microsoft.Ml.Samples project source files. 
1- Remove the copyright from the docs/Microsoft.Ml.Samples project sources.
2- Change the other source files files that reference the samples in the XML to not contain a range
. like [here](https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.FastTree/TreeTrainersStatic.cs#L41): 

the reference to the other file is:

`[!code-csharp[FastTree](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Static/FastTreeRegression.cs?range=6-11,19-69 "FastTree regression example.")]
`
and it should change to not include a range nor comment anymore:

`[!code-csharp[FastTree](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Static/FastTreeRegression.cs)]
`

### PR DESCRIPTION
**Pull request by Caraul**
Fixes #1576
Remove the copyright from all docs/Microsoft.Ml.Samples project sources including Program.cs and correct !code-csharp references removing ranges and comments.
Left Timeseries.cs as is due to #1577 and #1578 and range references to its content.

### CODE REVIEW

**Comment by sfilipi** [comment]
>         using Microsoft.ML.Transforms;

can you please remove the tabs from the left side of the usings. 

With the copyright removed we can reference the full file now. 
Same thing for all the other files. 

---
Refers to: docs/samples/Microsoft.ML.Samples/Dynamic/ConcatTransform.cs:7 in c0212f1. [](commit_id = c0212f1380e18428c4f19f40652f2ace93f00923, deletion_comment = False)

**Comment by sfilipi** [comment]
>     public partial class TransformSamples

nit make them not partial, and call it something like "ConcatExample, so they are copy/pasted and don't need any modification. 

---
Refers to: docs/samples/Microsoft.ML.Samples/Dynamic/ConcatTransform.cs:11 in c0212f1. [](commit_id = c0212f1380e18428c4f19f40652f2ace93f00923, deletion_comment = False)

**Comment by sfilipi** [comment]
>         // the alignment of the usings with the methods is intentional so they can display on the same level in the docs site.

this comment can go away too. 

---
Refers to: docs/samples/Microsoft.ML.Samples/Dynamic/ConcatTransform.cs:1 in c0212f1. [](commit_id = c0212f1380e18428c4f19f40652f2ace93f00923, deletion_comment = False)

File: src/Microsoft.ML.TimeSeries/IidChangePointDetector.cs [hunk 1]
```diff
@@ -204,7 +204,7 @@ public sealed class IidChangePointEstimator : TrivialEstimator<IidChangePointDet
         /// <example>
         /// <format type="text/markdown">
         /// <![CDATA[
-        /// [!code-csharp[MF](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Timeseries.cs?range=6-12,17-37,231-293
```
Comment (sfilipi) [inline src/Microsoft.ML.TimeSeries/IidChangePointDetector.cs]: >c/Timeseries.cs?range=6-12,17-37,231-293 [](start = 88, length = 40)

please leave the ranges in the TImeSeries files. The samples on the time series are all in one file, that's why we still need to reference them by region.  Issue #1577 has an ask to split them up; than we can reference them by file name only.

**Comment by sfilipi** [comment]
thanks for the help @Caraul. Heads up that PR #1578 is doing the same thing; work not completed yet. 

---
In reply to: [438777431](https://github.com/dotnet/machinelearning/pull/1619#issuecomment-438777431) [](ancestors = 438777431)

**Comment by Caraul** [comment]
2018-11-15T07:05:25.3681590Z ##[section]Starting: Run Tests
...
2018-11-15T07:44:09.1885500Z ##[error]The operation was canceled.
2018-11-15T07:44:09.1903920Z ##[section]Finishing: Run Tests

##[Error 1]
The job running on agent Hosted macOS 4 has exceeded the maximum execution time of 45. 

On MacOS only - what should I do in this case? Looks like problem isn't in tests.

**Comment by sfilipi** [comment]
we do have some build glitches. Just re-trigger the build. I'll keep an eye too. 

---
In reply to: [439013327](https://github.com/dotnet/machinelearning/pull/1619#issuecomment-439013327) [](ancestors = 439013327)

**Comment by sfilipi** [comment]
>     public partial class TrainerSamples

public class MatrixFactorizationSample

---
Refers to: docs/samples/Microsoft.ML.Samples/Dynamic/MatrixFactorization.cs:12 in 77bb89a. [](commit_id = 77bb89af7a1bdf3de1608fc364522ad3364fdab6, deletion_comment = False)

**Comment by Caraul** [comment]
> > ```
> > public partial class TrainerSamples
> > ```
> 
> public class MatrixFactorizationSample
> 
> Refers to: docs/samples/Microsoft.ML.Samples/Dynamic/MatrixFactorization.cs:12 in [77bb89a](https://github.com/dotnet/machinelearning/commit/77bb89af7a1bdf3de1608fc364522ad3364fdab6). [](commit_id = [77bb89a](https://github.com/dotnet/machinelearning/commit/77bb89af7a1bdf3de1608fc364522ad3364fdab6), deletion_comment = False)

My fault - add small fix for it. I also suppose "// NOTE: WHEN ADDING TO THE FILE, ALWAYS APPEND TO THE END OF IT." warning is obsoleted as there are no more range reference - am I correct?

**Comment by sfilipi** [comment]
> > > ```
> > > public partial class TrainerSamples
> > > ```
> > 
> > 
> > public class MatrixFactorizationSample
> > Refers to: docs/samples/Microsoft.ML.Samples/Dynamic/MatrixFactorization.cs:12 in [77bb89a](https://github.com/dotnet/machinelearning/commit/77bb89af7a1bdf3de1608fc364522ad3364fdab6). [](commit_id = [77bb89a](https://github.com/dotnet/machinelearning/commit/77bb89af7a1bdf3de1608fc364522ad3364fdab6), deletion_comment = False)
> 
> My fault - add small fix for it. I also suppose "// NOTE: WHEN ADDING TO THE FILE, ALWAYS APPEND TO THE END OF IT." warning is obsoleted as there are no more range reference - am I correct?

Yes, it is obsolete. Thanks for taking care of it.

File: docs/samples/Microsoft.ML.Samples/Static/FastTreeRegression.cs [hunk 2]
```diff
@@ -1,20 +1,15 @@
...
-        using System;
-        using System.Linq;
+﻿using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Trainers.FastTree;
+using Microsoft.ML.StaticPipe;
+using System;
+using System.Linq;
 
 // NOTE: WHEN ADDING TO THE FILE, ALWAYS APPEND TO THE END OF IT. 
 // If you change the existinc content, check that the files referencing it in the XML documentation are still correct, as they reference
```
Comment (justinormont) [inline docs/samples/Microsoft.ML.Samples/Static/FastTreeRegression.cs]:
```suggestion
// If you change the existing content, check that the files referencing it in the XML documentation are still correct, as they reference
```

Comment (Caraul) [reply docs/samples/Microsoft.ML.Samples/Static/FastTreeRegression.cs]: Above we already discussed these comments as obsoleted - would it be better to completely remove them from samples? I may do that in this same pull request.

**Pull request merged**